Release upload failed due to 'ContextualVersionConflict: urllib3 1.22'

(venv) vagrant@vagrant-ubuntu-trusty-64:/vagrant$ make upload
==> This will upload zhmc-ansible-modules version 0.2.0 to PyPI!
==> Continue? [yN] y
twine upload build/dist/zhmc_ansible_modules-0.2.0-py2.py3-none-any.whl build/dist/zhmc-ansible-modules-0.2.0.tar.gz
ContextualVersionConflict: urllib3 1.22
make: *** [upload] Error 1

--- a/requirements.txt
+++ b/requirements.txt
@@ -44,5 +44,5 @@ requests>=2.12.4,!=2.17.1,!=2.17.2 # Apache-2.0, from zhmcclient
 six>=1.10.0 # MIT, from zhmcclient and others
 stomp.py>=4.1.15 # Apache, from zhmcclient
 tabulate>=0.7.7 # MIT, from zhmcclient
-urllib3>=1.21.1 # MIT, from requests
+urllib3>=1.21.1,<1.22 # MIT, from requests
 wcwidth>=0.1.7 # MIT, from prompt_toolkit

